.github/workflows/ci.yml

```diff
@@ -94,9 +94,17 @@ jobs:
       # fix for forcing git to use https when pulling deps
       - run: 'git config --global --replace-all url."https://github.com/".insteadOf ssh://git@github.com/'
       - run: npm ci
-      - run: npm run cy:coverage
+      - name: stateful tests
```

Comment:
these steps will run back to back - it may be better to define a separate job for these so they can each have their own machine. If we do this then we won't necessarily need to use the `parallel` flag since we're basically manually parallelizing